inspector/scripts/inspect-tbtc.js

```diff
@@ -0,0 +1,170 @@
+const DepositJson = require("@keep-network/tbtc/artifacts/Deposit.json")
+const DepositFactoryJson = require("@keep-network/tbtc/artifacts/DepositFactory.json")
+const BondedECDSAKeepJson = require("@keep-network/keep-ecdsa/artifacts/BondedECDSAKeep.json")
+
+const truffleContract = require("@truffle/contract")
+
+const deploymentBlock = 10867764
```

Comment:
Can we read it from JSON? If we keep it hardcoded inspecting testnet becomes problematic.

Reply:
Done.